prow/config/tide.go

```diff
@@ -80,6 +80,11 @@ type Tide struct {
 	// the default method of merge. Valid options are squash, rebase, and merge.
 	MergeType map[string]github.PullRequestMergeType `json:"merge_method,omitempty"`
 
+	// MergeApproveTag is the tag appended to the automatic commit
+	// message for each approver.  A recommended value (if you want this
+	// feature at all) is Reviewed-by.
```

Comment:
imo we should reword slightly to clarify that an empty value disables the feature.

Reply:
> imo we should reword slightly to clarify that an empty value disables the feature.

Done in b0c751d -> 97f6fa5.